### PR TITLE
Add util for modify decimal numbers

### PR DESCRIPTION
**Pull request by hectoruch**

### CODE REVIEW

**Comment by nerik** [comment]
It is partially a duplicate of https://github.com/Vizzuality/TRASE-frontend/blob/develop/scripts/utils/niceNumber.js

Can you merge the two? Always use `toLocaleString` for the end result because it properly shows the decimal separator depending on locale (, for US, . for Spain)

File: scripts/utils/changeNumber.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        numberChange = number.toFixed(10);
+      } else {
+        numberChange = 0;
+      }
+    }
+  } else{ // if is null
+    numberChange = 0;
+  }
+
+  return numberChange.toLocaleString();
```
Comment (agnlez) [inline scripts/utils/changeNumber.js:22]: Why `String` ?

Comment (agnlez) [reply scripts/utils/changeNumber.js:22]: Didn't notice about @nerik advice. Ignore this.

**Comment by hectoruch** [comment]
please @andresgnlez review it.

**Pull request merged**